config/electronBuilder.config.js

```diff
@@ -45,6 +45,12 @@ module.exports = {
         notarize: {
             teamId: '368M544MTT',
         },
+        target: [
+            {
+                target: 'dmg',
+                arch: ['universal'],
+            },
+        ],
```

Comment:
Changing the target to `dmg` broke app updates. From the [electron-builder docs](https://github.com/electron-userland/electron-builder/blob/81da7c1491e2b3ff2f7b476f8128183f57074ff5/docs/configuration/mac.md): Squirrel.Mac auto update mechanism requires both `dmg` and `zip` to be enabled, even when only `dmg` is used. Disabling `zip` will break auto update in dmg packages.

We should have left the target set to `default`